modules/p2-profile-gen/pom.xml

```diff
@@ -527,18 +541,428 @@
 									<id>org.wso2.carbon.identity.workflow.mgt.feature.group</id>
 									<version>${carbon.identity.version}</version>
 								</feature>
-                                <feature>
+                                				<feature>
+									<id>org.wso2.ciphertool.feature.group</id>
+									<version>${cipher-tool.version}</version>
+								</feature>
+								<feature>
+								    <id>org.wso2.carbon.dashboards.shindig.feature.group</id>
+								    <version>${carbon.dashboards.version}</version>
+								</feature>
+								<feature>
+									<id>org.wso2.carbon.user.mgt.workflow.server.feature.group</id>
+									<version>${carbon.identity.version}</version>
+								</feature>
+							</features>
+						</configuration>
+					</execution>
+					<execution>
+                        	<id>publishing product</id>
+                        			<phase>package</phase>
+                        			<goals>
+                            				<goal>publish-product</goal>
+                        			</goals>
+                        			<configuration>
+                            				<productConfigurationFile>${basedir}/carbon.product
+                            				</productConfigurationFile>
+                            				<executable>
+                                				${basedir}/target/org.eclipse.equinox.executable_3.5.0.v20110530-7P7NFUFFLWUl76mart
+                           			        </executable>
+                            				<metadataRepository>file:${basedir}/target/p2-repo</metadataRepository>
+                            				<artifactRepository>file:${basedir}/target/p2-repo</artifactRepository>
+                        			</configuration>
+                    			</execution>
+
+			 		<execution>
+						<id>creating-workflow-profile</id>
+						<phase>package</phase>
+						<goals>
+						    <goal>materialize-product</goal>
+						</goals>
+						<configuration>
+							<productConfigurationFile>${basedir}/carbon.product</productConfigurationFile>
+							<metadataRepository>file:${basedir}/target/p2-repo</metadataRepository>
+							<artifactRepository>file:${basedir}/target/p2-repo</artifactRepository>
+							<targetPath>
+								file:${basedir}/target/wso2carbon-core-${carbon.kernel.version}/repository/components
+							</targetPath>
+							<profile>workflow</profile>
+						</configuration>
+					    </execution>
+
+
+					<execution>
+						<id>p2-profile-generation-for-workflow</id>
+						<phase>package</phase>
+						<goals>
+							<goal>p2-profile-gen</goal>
+						</goals>
+						<configuration>
+							<profile>workflow</profile>
```

Comment:
Looks like workflow profile starts with all the bundles. How do we start with only BPS bundles, like a BPS backend server ? Can we add another profile ? I would suggest lets have 3 profiles.
1. default - carbon-identity bundles only
2. workflow - business-process bundles only
3. full - all bundles